controllers/tasks/replication/enableReplication.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package replication
```

Comment:
since the package name is `replication`,  we could remove word `replication` from the file name and in the struct name. 
`enable.go`

``` 
type EnableTask struct {
}```